bundled/tool/server.py

```diff
@@ -11,6 +11,11 @@
 import traceback
 from typing import Any, Callable, Dict, List, Optional, Sequence, Union
 
+from pygls import lsp, protocol, server, uris, workspace
+
+import jsonrpc
+import utils
+
```

Comment:
@jjmacklin This needs to go back to line 37, the code won't run otherwise. Do let me know if you have issue, I can help with this.

Reply:
@karthiknadig Ok, I finally realized what the issue was. Every time I saved, it was autoformatting the imports to the top of the file, and I didn't notice until after the push. I disabled that in vscode settings, and it should be fine now. Thanks for your patience!